specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -7822,6 +7822,162 @@
       },
       "description": "Text analytics positive-negative sentiment analysis, scored as a floating point value in a range of zero to 1."
     },
+    "SentimentSkillV3": {
```

Comment:
Do we prefer `SentimentSkillV3` or '`SentimentSkill3` here?
In the few places in the .NET SDK where we have had to use version numbers in the types, I see we use the latter -
- [IdentityAutoGenerated2](https://adpnetsource.azurewebsites.net/#Azure.ResourceManager.Resources/Generated/Models/IdentityAutoGenerated2.cs,856bc5ce577027a5)
- [CloudErrorAutoGenerated2](https://adpnetsource.azurewebsites.net/#Azure.ResourceManager.Storage/Generated/Models/CloudErrorAutoGenerated2.cs,0e7932a17dcdab5c)

Reply:
There  is going to be custom logic implemented for these two V3 skills in the SDK, so I'm not sure it matters. @alzimmermsft is leading those efforts, discussed here: https://github.com/Azure/azure-sdk/issues/2987